### DISCUSSION TITLE
Are you seeking contributors?

### DISCUSSION THREAD

**Discussion by WheelieDev** [vitodeploy/vito#53] [General]
Hello, love the project. I think deployment can be one of the highest learning curves and this is a great tool to address that. Are you seeking contributors at the moment? Would love to look at the docs and convert the User create command from arguments to an interactive wizard as first steps

**Comment by saeedvaziry** [comment]
Hi, Feel free to open a PR. The priority is always the issues but any contribution is welcome :) So feel free with the wizard steps you mentioned.
